core/blockchain/impl/block_tree_impl.hpp

```diff
@@ -274,17 +274,16 @@ namespace kagome::blockchain {
 
     } block_tree_data_;
 
-    primitives::events::ExtrinsicSubscriptionEnginePtr
-        extrinsic_events_engine_ = {};
-    primitives::events::ChainSubscriptionEnginePtr chain_events_engine_ = {};
+    primitives::events::ChainSubscriptionEnginePtr chain_events_engine_;
+    std::shared_ptr<PoolHandler> main_pool_handler_;
```

Comment:
This member added, but not used (no changes). Looks unused

Reply:
This fixes warning about constructor fields initialization order.